com.tonikelope.MegaBasterd.yaml

```diff
@@ -0,0 +1,59 @@
+app-id: com.tonikelope.MegaBasterd
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: wrapper.sh
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --env=PATH=/app/jre/bin:/app/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+  - --filesystem=xdg-download
+  - --persist=.
```

Comment:
I'm not sure about this? why does it need it?

Reply:
Because it stores its config files in versioned directories inside the home folder, e.g. MegaBasterd 7.28 uses `~/.megabasterd7.28`, while 7.29 will use `~/.megabasterd7.29`. So, when you launch the program after an upgrade, it asks you if you want to migrate your old config folder to the new one. Then, a new backup folder called `~/.megabasterd_old_backups` is also created, containing config files of previous versions.